src/Simplex/Chat/Migrations/M20241027_server_operators.hs

```diff
@@ -0,0 +1,55 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Migrations.M20241027_server_operators where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20241027_server_operators :: Query
+m20241027_server_operators =
+  [sql|
+CREATE TABLE server_operators (
+  server_operator_id INTEGER PRIMARY KEY AUTOINCREMENT,
+  server_operator_tag TEXT,
+  trade_name TEXT NOT NULL,
+  legal_name TEXT,
+  enabled INTEGER NOT NULL DEFAULT 1,
+  role_storage INTEGER NOT NULL DEFAULT 1,
+  role_proxy INTEGER NOT NULL DEFAULT 1
+);
+
+ALTER TABLE protocol_servers ADD COLUMN server_operator_id INTEGER REFERENCES server_operators ON DELETE SET NULL;
+
+CREATE TABLE usage_conditions_acceptance (
```

Comment:
why change from previous many-to-many schema? I think it made sense:

adding specific conditions (without text but with commit) only once, linking operators to that single record